Cargo.toml

```diff
@@ -21,24 +21,21 @@ is-it-maintained-open-issues = { repository = "mozilla/mtu", branch = "main" }
 maintenance = { status = "actively-developed", branch = "main" }
 
 [dependencies]
-# Don't increase beyond what Firefox is currently using: https://searchfox.org/mozilla-central/source/Cargo.lock
-libc = { version = "0.2", default-features = false }
-static_assertions = { version = "1.1", default-features = false }
+libc = { version = "0", default-features = false }
+static_assertions = { version = "1", default-features = false }
 
 [target.'cfg(windows)'.dependencies]
-# Don't increase beyond what Firefox is currently using: https://searchfox.org/mozilla-central/source/Cargo.lock
-windows = { version = ">=0.58,<0.60", features = [
+windows = { version = "0", features = [
```

Comment:
I don't think `0` is a good idea here. I am not even sure it is a valid version.

We know `mtu` works with `windows` `0.58` and `0.59`, thus the above range. I suggest keeping that exact range here. In case we know `mtu` works with e.g. `windows `0.6`, then we can extend the range.